src/zoid/buttons/component.jsx

```diff
@@ -610,6 +610,13 @@ export const getButtonsComponent: () => ButtonsComponent = memoize(() => {
         value: getIntent,
       },
 
+      jsSdkLibrary: {
+        type: "string",
+        queryParam: true,
+        required: false,
```

Comment:
is `required: false` the default value? just curious. but also the `type: "string"` could come after `required`